definitions/procedures/selinux/set_file_security.rb

```diff
@@ -7,6 +7,9 @@ class SetFileSecurity < ForemanMaintain::Procedure
             'Is the backup incremental?',
             :required => true
       manual_detection
+      confine do
+        package_manager.installed?(['policycoreutils'])
```

Comment:
Should this instead check for existence if `/sys/fs/selinux`? IMHO cleaner than checking for a package.